python/plugins/processing/modeler/ModelerParametersDialog.py

```diff
@@ -360,9 +360,10 @@ def getWidgetFromParameter(self, param):
             else:
                 item = QComboBox()
                 item.setEditable(True)
+                item.addItem(unicode(param.default or "")+" (the default)", param.default)
```

Comment:
Yes, the " (the default)" or whatever it should be, should be wrapped in self.tr()